data_model/src/events/mod.rs

```diff
@@ -76,16 +70,11 @@ impl VersionedEventSubscriberMessage {
     }
 }
 
-/// Message sent by the stream consumer
+/// Message sent by the stream consumer.
+/// Request sent by the client to subscribe to events.
 #[version_with_scale(n = 1, versioned = "VersionedEventSubscriberMessage")]
-#[derive(Debug, Clone, Decode, Encode, FromVariant, IntoSchema)]
-pub enum EventSubscriberMessage {
-    /// Request sent by the client to subscribe to events.
-    //TODO: Sign request?
-    SubscriptionRequest(FilterBox),
-    /// Acknowledgment of receiving event sent from the peer.
-    EventReceived,
-}
+#[derive(Debug, Clone, Decode, Encode, IntoSchema)]
+pub struct EventSubscriberMessage(pub FilterBox);
```

Comment:
consider renaming to `SubscriptionRequest`

Reply:
I will use `EventSubscriptionRequest`